scalatags/src/scalatags/generic/Bundle.scala

```diff
@@ -102,53 +104,54 @@ trait Aliases[Builder, Output <: FragT, FragT]{
 
   type Tag = generic.TypedTag[Builder, Output, FragT]
 
+  // Scala 3 behaviour prevents us from using the same name as the case
+  // class for some reason.
+
   /**
    * A [[Modifier]] which contains a String which will not be escaped.
    */
   protected[this] type RawFrag <: Modifier
-  protected[this] val RawFrag: Companion[RawFrag]
```

Comment:
I removed this and all the tests pass.
The way unapply works in Scala 3 makes it hard to write it generically.
Does this have an usage @lihaoyi ?

Reply:
I don't think this is generally used. People construct raw frags via `raw(...)`, and generally we do not support or encourage de-constructing frags: they are to be constructed and rendered, with no other operations in between. 

Thus we can probably do without this if it's too difficult. If removing it doesn't break the test suite, then we it probably won't be missed